app/templates/components/event-card.hbs

```diff
@@ -32,9 +32,13 @@
         <i role="button" class="share alternate link icon" {{action shareEvent event}}></i>
       </span>
       <span>
-        {{#each tags as |tag|}}
-          <a>{{tag}}</a>
-        {{/each}}
+        {{#if isYield}}
```

Comment:
@niranjan94 you mean to say, don't pass anything while calling the component and directly check if there is something to yield via 
```
{{# if yield}}
  {{yield}}
{{else}}
  ...
{{/if}}